### ISSUE TITLE
Hotel Mode Booked days inclusivity problem

### ISSUE THREAD

**Issue by sansobarcelo** [wakirin/Litepicker#30]
Hi,
First of all thanks for this Calendar, it is perfect.

Working with the calendar I found a small problem.

When I set the calendar to hotel mode, bookedDaysInclusivity: '()', and setting bookedDays, see the initialitzation below
```javascript
new Litepicker({
            element: dateInputRef.current,
            format: dateformat,
            hotelMode: true,
            minDays: 1,
            disallowBookedDaysInRange: true,
            selectForward: true,
            bookedDaysInclusivity: '()',
            singleMode: false,
            onSelect: onRangeSelected,
        });
```
When I have two bookings, for example: ['2020/02/06', '2020/02/14'] and ['2020/02/14', '2020/02/20'], the day 14 should not be available because in this day we already have a checkin and a checkout.

See the result:

![image](https://user-images.githubusercontent.com/5562806/72414250-1537cf80-3772-11ea-8401-1d02852622a1.png)

Is there any way to solve this? It is not a big problem, because disabling the option to pick just one day, the functional problem is solved.

Thanks

**Comment by myraumhafen** [comment]
Hello everyone, I can confirm that problem. I noticed that a few days ago. If check-out and check-in fall on the same day, the day is available in the selection. I solved this for myself by checking the days booked in advance and summarizing the days booked in the array. Then ['2020/02/06', '2020/02/14'] and ['2020/02/14', '2020/02/20'] will become ['2020/02/06', '2020/02/20'].
Greetings, Michael

**Comment by sansobarcelo** [comment]
> Hello everyone, I can confirm that problem. I noticed that a few days ago. If check-out and check-in fall on the same day, the day is available in the selection. I solved this for myself by checking the days booked in advance and summarizing the days booked in the array. So ['2020/02/06', '2020/02/14'] and ['2020/02/14', '2020/02/20'] will become ['2020/02/06', '2020/02/20'].
> Greetings, Michael

Yes, that's the workaround that I've applied!

**Comment by wakirin** [comment]
Fixed in v1.0.29
https://jsfiddle.net/dmLx4os8/
